### ISSUE TITLE
Update Salsah dependencies

### ISSUE THREAD

**Issue by janCstoffregen** [dhlab-basel/Salsah#242]
Dear André,
Salsah can't be set up at the moment because of the change in the npm - version:
```
Cloning into 'Salsah'...
Removing intermediate container b1fd7f341e7f
 ---> 63b36182ef4e
Step 14/17 : RUN cd /Salsah && yarn install --production=false
 ---> Running in 26ed3d1fe8fe
yarn install v1.6.0
[1/4] Resolving packages...
[2/4] Fetching packages...
error An unexpected error occurred: "https://registry.yarnpkg.com/@salsah/progress-indicator/-/progress-indicator-1.0.0-beta.3.tgz: Request failed \"404 Not Found\"".
```

Best regards,
Jan

**Comment by kilchenmann** [comment]
Yes, you're right. I will fix this in the afternoon today

**Comment by janCstoffregen** [comment]
ok, thanks!

**Comment by kilchenmann** [comment]
This issue should be fixed with PR #243. Can you make a review, so I can merge it back into develop.

**Comment by janCstoffregen** [comment]
Hi, thank you so much, I'll check it now
